addon/-private/system/model/internal-model.js

```diff
@@ -885,17 +885,28 @@ export default class InternalModel {
     @method clearRelationships
     @private
   */
-  clearRelationships() {
+  clearRelationships(resetAll = false) {
```

Comment:
we may not need to split the methods, depending on the outcome of our discussion -> https://github.com/emberjs/data/pull/5058#discussion_r126597331